sensibility/miner/downloader.py

```diff
@@ -200,12 +202,15 @@ def fetch_repository(self, repo: RepositoryID) -> RepositoryMetadata:
             raise ValueError(f'Could not fetch info for {repo}')
         owner, name = RepositoryID.parse(info['nameWithOwner'])
         latest_commit = info['defaultBranchRef']['target']
+        license_name: Optional[str] = None
```

Comment:
After reading that documentation again, I realize that the License.name is declared as `String!`, meaning that info['licenseInfo']['name'] is **MUST** be a string! So you don't need to check set it as `None`, or check if it's `None`... In theory.

Reply:
Hi @eddieantonio, you are right, however, in line `214` it takes an optional string for the field `license`.

```Python
...
license=license_name,
...
```

See that `name` in the `licenseInfo` always MUST be a string!, but the field `licenseInfo` is optional, meaning that I have an optional `license_name` as it only must exist if a `licenseInfo` is __not__ `None`

Reply:
Ah, okay. Apparently `RepositoryMetadata.license` **MUST** be a string (for reasons I can't quite remember), so you may want to change line 214 to this:

```python
license=license_name or '',
```

So if `license_name` still ends up being `None`, it is set as the empty string as a fallback. This should still type-check.

Reply:
Another thing is that I was having a crash (please see `commit dffdccded26094b44c2dfad85841fbf861e26f9b - Fixing potencial bug as the licenseInfo is an optional value`)

will make the change on line `214` you requested!